CHANGELOG.md

```diff
@@ -14,6 +14,17 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 :microscope: - experimental
 
+## [2.5.0]
+Breaking change:
+- :rocket: added step `I will wait for alert/dialog`
+
+```gherkin
+  Scenario: accept alert
+    Given I will wait for dialog
```

Comment:
To be honest it is not obvious what this step is for, future tesnse confuses a bit.
Lets consider 2 other solutions:
1) Rename the step to `Given I set alert listener`
or more elegant solution
2) Using BeforeAll hook check if some of pickles include 'alert' keyword and  then set the listener if it have not been alredy set.

Reply:
For me `listener` is more programming specific. I want to use this `I will wait` for other types of event emitters available in playwright

Reply:
Given I expect alert will appear in this scenario

Reply:
To be honest I don't really like to make use of steps that dos not represent a business logic.
I know, this is not a silver bullet last option is adding a listener using a tag expression.

Reply:
I agree, but making it implicit will add more constraints and increase support cost